### PR TITLE
APE-1311 | add customer tracking headers for AIS calls for 'app' sdk

### PR DESCRIPTION
**Pull request by hitendra-ap**

### CODE REVIEW

**Review by JulesPatry — APPROVED** [review]

**Review by sibinlu — APPROVED** [review]

File: app/src/main/Member.js [hunk 1]
```diff
@@ -823,6 +824,27 @@ export default class Member extends CoreMember {
...
+     * Populate customer tracking metadata headers in token calls.
+     *
+     * @param {CustomerTrackingMetadata} customerTrackingMetadata
+     * @return new member
+     */
+    addCustomerTrackingMetadata(
+        customerTrackingMetadata: CustomerTrackingMetadata,
+    ): Member {
+        return Util.callSync(this.addCustomerTrackingMetadata, () => {
+            const newMember = new Member(this._options);
```
Comment (sibinlu) [inline app/src/main/Member.js:837]: do we need to create a new member?
`addCustomerTrackingMetadata` sounds more like changing an existing member

Comment (hitendra-ap) [reply app/src/main/Member.js]: removed creation of a new member

**Comment by JugrajSripalJain** [comment]
@hitendra-ap who approved the PR?

**Comment by hitendra-ap** [comment]
@JugrajSripalJain I asked @sibinlu for the go-ahead

**Pull request merged**